tests/robot/load_variables_from_profiles.py

```diff
@@ -0,0 +1,33 @@
+#
+#    Copyright 2017 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Files structure reflects the same in legion. I can't see big benefit of rearranging the methods, instead of benefits this may lead to confusion I guess. Also this layout looks quite logical as for me - profile_loader seems to be a more legion specific stuff while the load_variables_from_profiles is a kind of entry point.
Does it make sanse as for you?